food_ratings/frontend/views.py

```diff
@@ -1,4 +1,6 @@
-import requests
+# -*- coding: utf-8 -*-
+import requests, sys, os
+sys.path.append('..')
```

Comment:
why are you doing this? it's not obvious what this line is for and it looks suspicious.